benchmarks/high_volume_writes_test.go

```diff
@@ -50,69 +57,55 @@ var _ = Describe("Scenario: High Volume Writes", func() {
 
 		if totalSamples == 0 {
 			Expect(logger.Undeploy(k8sClient, benchCfg.Logger)).Should(Succeed(), "Failed to delete logger deployment")
+			configurationCount++
+			if configurationCount >= len(scenarioCfg.Configurations) {
+				return
+			}
```

Comment:
Is this block doing any relevant work?

Reply:
It is just for sanity, No need for that code,  deleted

Reply:
done